code/modules/research/xenobiology/vatgrowing/petri_dish.dm

```diff
@@ -26,7 +37,7 @@
 	if(!sample || !istype(A, /obj/structure/sink))
 		return FALSE
 	to_chat(user, span_notice("You wash the sample out of [src]."))
-	sample = null
+	QDEL_NULL(sample)
```

Comment:
You should keep this as `sample = null`--this is a pointer to the original sample, and so you'll be deleting that too.

![image](https://user-images.githubusercontent.com/35135081/161139900-948249ea-ef54-40c7-b9f8-3fe45c8f2af5.png)

Reply:
Destroy and the growing vat's plunger_act already QDEL_NULL the sample. Also, are you sure there is an "original sample"? The deposited_sample here is created in the proc try_to_swab from the contents of swabbed_items, which are then cleared, and only stays in the dish. When the sample is deposited into a growing vat, that growing vat creates a copy of the dish's sample. One sample should only be in one place at a time.
![Przechwytywanie1](https://user-images.githubusercontent.com/102324362/161381751-6c846543-2317-4f6b-8d22-293b6f9f490a.PNG)
![Przechwytywanie](https://user-images.githubusercontent.com/102324362/161381348-b288e0ba-e8a1-4257-84e9-099eb220193f.PNG)

Reply:
Anyways the qdel wasn't really necessary to fix the bug so I removed it.